### PR TITLE
Update join plugin init params

### PR DESCRIPTION
**Pull request by roienatan**
resolves: https://github.com/daostack/arc.js/issues/541

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daostack/arc.js",
-  "version": "2.0.0-experimental.56",
+  "version": "2.0.0-experimental.57",
```
Comment (orenyodfat) [inline package.json:3]: no need to bump version

File: src/plugins/join/plugin.ts [hunk 2]
```diff
@@ -126,7 +124,7 @@ export class Join extends ProposalPlugin<
 
     const baseState = Plugin.itemMapToBaseState(context, item)
 
-    const fundingRequestParams = {
+    const joinParams = {
```
Comment (orenyodfat) [inline src/plugins/join/plugin.ts:127]: why fundingRequestParams replaced to joinParams ?

Comment (roienatan) [reply src/plugins/join/plugin.ts:127]: It's join plugin file, it's suppose to be joinParams.

File: src/plugins/join/plugin.ts [hunk 3]
```diff
@@ -92,7 +91,7 @@ export class Join extends ProposalPlugin<
     return this.fragmentField
   }
 
-  public static initializeParamsMap(initParams: IInitParamsJQ) {
+  public static initializeParamsMap(initParams: IInitParamsJ) {
```
Comment (orenyodfat) [inline src/plugins/join/plugin.ts:94]: how about call that `IInitParamsJoin` ?

**Review by orenyodfat — APPROVED** [review]

**Pull request merged**